{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/analytics/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Def need to flesh this out :)

You'd be surprised how often things break in the template, super high test coverage pays dividends